sched/scheduler/jobStatus.go

```diff
@@ -0,0 +1,83 @@
+package scheduler
+
+import (
+	s "github.com/scootdev/scoot/saga"
+	"github.com/scootdev/scoot/sched"
+)
+
+type JobStatus struct {
+	Id         string
+	Status     sched.Status
```

Comment:
Yeah, it's bugged me before, so I thought I'd mention it so we could figure out if there's a way to reduce confusion in both this specific and the general case, cause, darnit...